lib/utils/annotation.go

```diff
@@ -0,0 +1,36 @@
+package utils
+
+import (
+    "fmt"
+    metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+    "k8s.io/client-go/kubernetes"
+    "k8s.io/client-go/tools/clientcmd"
+    "utils"
+)
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+const TemplateAnnotation = "projectsveltos.io/template"
```

Comment:
Please just add a comment.

// This annotation is added on referenced ConfigMap/Secret to indicate their content is a template